src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
```

Comment:
I mean that I hope that we won't use this trick for something else. 

> sn't it just allocating one extra closure per full turn

no, per rejection. It is how @domenic's spec describes it, but this is an obvious optimisation.